spec/server/utils/configHelper.spec.js

```diff
@@ -0,0 +1,72 @@
+// Set the ENV before including the configHelper
+// otherwise they will not take effect.
+process.env.MONGODB_URL = 'foo';
+process.env.PORT_APP = '1234';
+process.env.PORT_DARKNET_MJPEG_STREAM = '1235';
+process.env.PORT_DARKNET_JSON_STREAM = '1236';
+
+const {
+  parseAndTestIsNumber,
+  getPortFromConfig,
+  getAppPort,
+  getJsonStreamPort,
+  getMjpegStreamPort,
+  getMongoUrl,
+} = require('../../../server/utils/configHelper');
+
+describe('configHelper', () => {
+  describe('parseAndTestIsNumber', () => {
+    it('should parse string to number', () => {
+      expect(parseAndTestIsNumber("123")).toBe(true);
+    });
+
+    it('should return false on non number s string', () => {
+      expect(parseAndTestIsNumber("abc")).toBe(false);
+    });
+
+    it('should return true when passed a number', () => {
+      expect(parseAndTestIsNumber(123)).toBe(true);
+    });
+  });
+
+  describe('getPortFromConfig', () => {
+    it('should return actual port by defined key', () => {
+      const config = {
+        PORTS: {
+          app: 8888,
+        },
+      };
+      expect(getPortFromConfig(config, 'app')).toBe(config.PORTS.app);
+    });
+
+    it('should return defined defaultPort because key does not exist', () => {
+      const config = {
+        PORTS: {
+          app: 1234,
+        },
+      };
+      expect(getPortFromConfig(config, 'foo', 3000)).toBe(3000);
+    });
+
+    it('should return defaultPort because key does not hold a number', () => {
+      const config = {
+        PORTS: {
+          app: 'ABC',
+        },
+      };
+
+      expect(getPortFromConfig(config, 'app', 3000)).toBe(3000);
+    });
+  });
+
+  describe('process env', () => {
+    // This test fails as soon as some other test suit includes the configHelper first they will no
+    // longer use the ENV values provided here but keep the other values
+    xit('takes ENV values by default', () => {
```

Comment:
maybe I can fix the test via Jasmine helpers?

Reply:
Fixed in https://github.com/opendatacam/opendatacam/pull/311/commits/01d87897723829ca05b0b97cdaa42bdb6d1ade2a